test/scapy/layers/eap.uts

```diff
@@ -60,9 +60,10 @@ assert eapol.haslayer(EAP_FAST)
 s = b'\x08\x02:\x01\x00\xc0\xcab\xa4\xf6\x00"k\xfbI+\x00"k\xfbI+\xa0[\xaa\xaa\x03\x00\x00\x00\x88\x8e\x02\x03\x00u\x02\x00\x8a\x00\x10\x00\x00\x00\x00\x00\x00\x00\x04\x95X{I5\':3\x8f\x90\xb1I\xae\x1f\xd7-"\x82\x1e\\$\xefC=\x83\x97?M\xd6\xdf>\x9b\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x16\xdd\x14\x00\x0f\xac\x04\x03\xca?d\xca\xed\xdd\xef\xf69;\xefX\xd4\x97w'
 wifi = Dot11(s)
 assert wifi[EAPOL].key_descriptor_type == 2
-assert wifi[EAPOL].key_type == 0
-assert wifi[EAPOL].has_key_mic == 1
-assert wifi[EAPOL].encrypted_key_data == 1
+assert wifi[EAPOL].encrypted_key_data == 0
+assert wifi[EAPOL].key_ack == 1
+assert wifi[EAPOL].key_type == 1
+assert wifi[EAPOL].key_descriptor_type_version == 2
```

Comment:
Looks like it matches `tshark` now:
```
    Key Information: 0x008a
        .... .... .... .010 = Key Descriptor Version: AES Cipher, HMAC-SHA1 MIC (2)
        .... .... .... 1... = Key Type: Pairwise Key
        .... .... ..00 .... = Key Index: 0
        .... .... .0.. .... = Install: Not set
        .... .... 1... .... = Key ACK: Set
        .... ...0 .... .... = Key MIC: Not set
        .... ..0. .... .... = Secure: Not set
        .... .0.. .... .... = Error: Not set
        .... 0... .... .... = Request: Not set
        ...0 .... .... .... = Encrypted Key Data: Not set
        ..0. .... .... .... = SMK Message: Not set
    Key Length: 16
    Replay Counter: 4
```

> RFC is kinda unclear

I'd say it's somewhat misleading. It just says
![Screenshot 2023-04-01 at 12 28 46](https://user-images.githubusercontent.com/10195800/229278379-1881820d-9449-4730-8d1d-cc41a11939b9.png)
and points to another standard where the bit order is mentioned.